### ISSUE TITLE
please add the time to the dates in the time series

### ISSUE THREAD

**Issue by wchristian** [CSSEGISandData/COVID-19#61]
Last i checked the time series files had both date AND time in the timestamps, but now it's down to only the dates. Can you please add the times again?

**Comment by iceweasel1** [comment]
I agree with you. I don't know if that's the intention. It's also interesting that when you add up the numbers from each day, the new data here doesn't match the older data at all. So I opened up a topic a few minutes ago. #64

**Comment by HyperBI** [comment]
i agree. If possible please bring back the time stamps. Sometimes its good to know when the updated time was.

**Comment by Sitius86** [comment]
They have added readme info to assert they are switching to date only going forward for the time series data. I didn't read it closely, not sure what their rationale was (possibly becoming overwhelming to manage with more and more countries reporting).

**Comment by InTEGr8or** [comment]
They also trimmed the leading `0` off the month and the day and reduced the year to a TWO DIGIT year, which we were all supposed to stop doing in 1999.

Whatever date format is selected, I just wish they'd quit switching it every few days.

**Comment by InTEGr8or** [comment]
Optimally, date-times for columns and file names would match, and be `yyyy-mm-dd`, plus `hh_mm` if they decide to include that. That will make the files sort in date order.

The current column nams `d/m/yy` require date-sorting and trimming the leading zeros when doing lookups in other documents. 

Changing date formats is a great way to cause unnecessary rework among everyone trying to use this data, so adopting the global best practice early would be a best practice in the long run, and it's looking like this might go for a long run.

**Comment by mingir2** [comment]
@InTEGr8or I agree with you. Having to write extra lines of code to convert their date format into the standard format is annoying to say the least.

**Comment by Bost** [comment]
https://github.com/CSSEGISandData/COVID-19/issues/238
